rent_golem/backend/rent_golem/clusters/models.py

```diff
@@ -18,10 +19,12 @@ class Status(models.TextChoices):
         TERMINATED = 'Terminated'
 
     uuid = models.UUIDField(primary_key=True)
+    owner = models.ForeignKey(User, on_delete=models.PROTECT, default=None)
     package_type = models.CharField(max_length=255, choices=Package.choices, default=Package.JUPYTER)
     status = models.CharField(max_length=255, choices=Status.choices, default=Status.STARTING)
     # https://stackoverflow.com/questions/67469569/using-django-jsonfield-in-model -> possible JSON Schema
     additional_params = models.JSONField()
     size = models.PositiveIntegerField()
     created_at = models.DateTimeField(auto_now_add=True)
     last_update = models.DateTimeField(auto_now=True)
+
```

Comment:
After adding is_delete flag, please add one test for each API method to check if deleted cluster cannot be: listed, retrieved, updated, deleted.